components/omega/src/ocn/TendencyTerms.h

```diff
@@ -271,6 +271,152 @@ class VelocityHyperDiffOnEdge {
    Array2DR8 EdgeMask;
 };
 
+// Tracer horizontal advection term
+class TracerHorzAdvOnCell {
+ public:
+   bool Enabled;
```

Comment:
Based on your previous comments, I believe turning on/off options of the tracer tendency terms will also be added to the default Config file in the next PR?

Reply:
Correct, the actual reading of the config file for these terms will occur alongside the config options for the other tendency terms, so I'll handle that when I integrate the tracer tendencies with the rest of the code.